test/runtests.jl

```diff
@@ -1,9 +1,6 @@
-using AlgebraicStockFlow
+using StockFlow
 using Test
-#using OrdinaryDiffEq
-
 
 @testset "AlgebraicStockFlow.jl" begin
-    # Write your tests here.
-    
+    @test true
```

Comment:
This is just a skeleton to test the test running, until we add an actual test suite.